### PR TITLE
fix for menu on Horz scroll

### PR DESCRIPTION
**Pull request by inexorableAce**
this fixes the horz scroll issues with the menu in #4164
ezgif com-video-to-gif (8)

### CODE REVIEW

**Comment by delphiactual** [comment]
works, but what happened that updated `yarn.lock`

**Comment by inexorableAce** [comment]
I dunno, I did yarn install and it touched yarn lock. Should it be ignored?

**Comment by delphiactual** [comment]
nope don't ignore it.

**Comment by delphiactual** [comment]
lol its bc GreenKeeper https://github.com/DestinyItemManager/DIM/pull/4181 does not update the `yarn.lock`

File: src/app/shell/header.scss [hunk 1]
```diff
@@ -15,10 +15,12 @@
     filter: hue-rotate(160deg) brightness(107%);
   }
 }
-
+body {
+  padding-top: 44px;
```
Comment (delphiactual) [inline src/app/shell/header.scss]:
```suggestion
  padding-top: $header_height;
```

Comment (inexorableAce) [reply src/app/shell/header.scss]: We have a header height var!? Nice! That’s much better

**Pull request merged**